wiki/Beatmap/Category/id.md

```diff
@@ -1,59 +1,51 @@
+
+## Sejarah
 
 ### Approved
 
-Approved merupakan status beatmap yang dahulu dipergunakan dari tahun 2008 hingga 2014. Status ini disematkan kepada beatmap-beatmap yang melebihi batas skor maksimum yang dipersyaratkan oleh [Ranking Criteria](/wiki/Ranking_Criteria) pada era *score ranking* di kala itu.
+Approved merupakan status beatmap yang dahulu dipergunakan dari tahun 2008 hingga 2014. Status ini disematkan kepada beatmap-beatmap yang melebihi batas skor maksimum dan durasi yang dipersyaratkan oleh [ranking criteria](/wiki/Ranking_Criteria) pada saat itu.
 
-Pada masanya dahulu, beatmap-beatmap Approved tidak memengaruhi total skor yang dimiliki oleh masing-masing pemain. Setelah osu! bertransisi dari sistem *score ranking* menuju ke sistem [*performance point ranking*](/wiki/Performance_points), status Approved kemudian dialihfungsikan untuk menandai beatmap-beatmap yang memiliki *[drain time](/wiki/Gameplay/Drain_time)* 6 menit atau lebih sebelum akhirnya perlahan dipensiunkan.
+Pada masanya dahulu, beatmap-beatmap Approved tidak memengaruhi total skor yang dimiliki oleh masing-masing pemain. Setelah osu! bertransisi dari sistem *score ranking* menuju ke sistem [*performance point ranking*](/wiki/Performance_points), status Approved kemudian dialihfungsikan untuk menandai beatmap-beatmap yang memiliki [*drain time*](/wiki/Gameplay/Drain_time) 6 menit atau lebih sebelum akhirnya perlahan dipensiunkan.
 
-Adapun beatmap-beatmap ini ditandai dengan simbol centang (![](/wiki/shared/status/approved.png)) yang dapat dilihat pada pojok kiri atas layar Song Selection.
+Adapun beatmap-beatmap ini ditandai dengan simbol centang (![](/wiki/shared/status/approved.png)) yang dapat dilihat pada pojok kiri atas layar song selection.
 
-### Loved
+### Kemunculan kategori Qualified
 
-*Untuk mengetahui lebih lanjut seputar latar belakang status Loved, harap kunjungi: [Sejarah Loved](/wiki/Beatmap/History_of_Loved)*
+Kategori [Qualified](#qualified) pertama kali diperkenalkan pada bulan Agustus 2014 saat [restrukturisasi BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) berjalan untuk membuat beatmap yang di-ranked lebih kelihatan. Jika beatmap dalam kategori Qualified perlu didiskualifikasi dan dikembalikan ke Pending, hal tersebut tidak akan mempengaruhi statistik pemain, karena kategori tersebut sudah didesain dari awal untuk tidak mencatat skor. Kemunculan kategori ini disambut dengan baik karena pemain dapat mencoba dan menentukan apakah map tersebut sudah memenuhi ekspektasi atau belum.
```

Comment:
`Kategori [Qualified](#qualified) pertama kali diperkenalkan pada bulan Agustus 2014 di tengah keberlangsungan [restrukturisasi BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat). Apabila suatu beatmap yang berstatus Qualified didiskualifikasi dan dikembalikan ke Pending, statistik para pemain tidak akan terpengaruh karena kategori Qualified sejak awal sudah dirancang untuk tidak mencatat skor-skor pemain secara permanen. Kemunculan kategori ini disambut dengan baik karena para pemain kini dapat mencoba dan menilai sendiri apakah suatu beatmap telah dirasa memenuhi standar atau belum.`